application/src/main/java/org/togetherjava/tjbot/commands/moderation/KickCommand.java

```diff
@@ -0,0 +1,108 @@
+        event.reply("I can not kick the given user since they are not part of the guild anymore.")
+            .setEphemeral(true)
+            .queue();
+    }
+
+    private static void kickUser(@NotNull Member target, @NotNull Member author,
+            @NotNull String reason, @NotNull Guild guild, @NotNull SlashCommandEvent event) {
+        User targetUser = target.getUser();
+        event.getJDA()
+            .openPrivateChannelById(targetUser.getId())
+            .flatMap(channel -> channel.sendMessage(
+                    """
+                            Hey there, sorry to tell you but unfortunately you have been kicked from the server %s.
+                            If you think this was a mistake, please contact a moderator or admin of the server.
+                            The reason for the kick is: %s
+                            """
+                        .formatted(guild.getName(), reason)))
+            .mapToResult()
+            .flatMap(sendDmResult -> {
+                logger.info("'{}' ({}) kicked the user '{}' ({}) from guild '{}' for reason '{}'.",
+                        author.getUser().getAsTag(), author.getId(), targetUser.getAsTag(),
+                        targetUser.getId(), guild.getName(), reason);
+
+                return guild.kick(target, reason)
+                    .reason(reason)
+                    .map(kickResult -> sendDmResult.isSuccess());
+            })
+            .map(hasSentDm -> {
+                String dmNotice =
+                        Boolean.TRUE.equals(hasSentDm) ? "" : "(Unable to send them a DM.)";
+                return ModerationUtils.createActionResponse(author.getUser(),
+                        ModerationUtils.Action.KICK, targetUser, dmNotice, reason);
+            })
+            .flatMap(event::replyEmbeds)
+            .queue();
+    }
+
+    @Override
+    public void onSlashCommand(@NotNull SlashCommandEvent event) {
+        Member target = Objects.requireNonNull(event.getOption(TARGET_OPTION), "The target is null")
+            .getAsMember();
+        Member author = Objects.requireNonNull(event.getMember(), "The author is null");
+        String reason = Objects.requireNonNull(event.getOption(REASON_OPTION), "The reason is null")
+            .getAsString();
+
+        Guild guild = Objects.requireNonNull(event.getGuild());
+        Member bot = guild.getSelfMember();
+
+        // Member doesn't exist if attempting to kick a user who is not part of the guild anymore.
+        if (target == null) {
+            handleAbsentTarget(event);
+            return;
+        }
+        if (!ModerationUtils.handleCanInteractWithTarget("kick", bot, author, target, event)) {
+            return;
+        }
+        if (!ModerationUtils.handleHasPermissions("kick", Permission.KICK_MEMBERS, bot, author,
+                guild, event)) {
+            return;
+        }
+        if (!ModerationUtils.handleReason(reason, event)) {
+            return;
+        }
```

Comment:
```suggestion
        if (target == null) {
            handleAbsentTarget(event);
            return;
        }
        
        if (!ModerationUtils.handleCanInteractWithTarget("kick", bot, author, target, event)) {
            return;
        }
        
        if (!ModerationUtils.handleHasPermissions("kick", Permission.KICK_MEMBERS, bot, author,
                guild, event)) {
            return;
        }
        
        if (!ModerationUtils.handleReason(reason, event)) {
            return;
        }
```
Be better if there was some space to make it easer to read

Reply:
Really? I disagree here. The spacing is there intentionally to separate the "read all variables", "check everything" and "actually kick" parts.

Reply:
If its all packed together it might make it hard to read.

Reply:
then i would rather introduce another helper method `handleChecks(...)` or something.

Reply:
Ok. will then its ok

Reply:
You can close this review. Its fine

Reply:
for the last 3 ifs, cant we just merge them with `||`? We don't need to worry about the other methods being run cause of short circuiting

Reply:
Not sure

Reply:
I think it should work. Try it out.